docs/contributing/documentation.rst

```diff
@@ -0,0 +1,49 @@
+.. _documentation:
```

Comment:
Add copyright and license, please

Reply:
Thanks for catching this! I suspect there will be more of these, I still need to go through it all at least once.